ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign.tsx

```diff
@@ -34,22 +36,23 @@ const TypedSignInfo: React.FC = () => {
   const {
     domain,
     domain: { verifyingContract },
-    primaryType,
   } = parseTypedDataMessage(currentConfirmation.msgParams.data as string);
 
+  const isPermitSigReq = isPermitSignatureRequest(currentConfirmation);
```

Comment:
Very minor, but should we avoid abbreviation in variables name for clarity? `isPermit`?